app/decorators/assignment_decorator.rb

```diff
@@ -22,6 +22,15 @@ def grading_scale_str
     end
   end
 
+  def restricted_features
+    restricted_feature_str = JSON.parse(assignment.feature_restrictions).reduce("") do |str, element|
```

Comment:
Why do we have to parse a jsonb column ?

Reply:
Ya right I guess we can just use reduce function on that as it supports indexing. Will change it sir.

Reply:
As to get implement to reduce function. Any other way sir?

Reply:
This is the output for Restricted feature if I do this `restricted_feature_str = assignment.feature_restrictions do |str, element|`

![bandicam 2022-01-09 12-48-46-071](https://user-images.githubusercontent.com/76901313/148673078-bf12c3ed-70f5-4118-acd0-929bf2015047.jpg)